resources/assets/js/components/UpdatingInput.tsx

```diff
@@ -76,7 +73,9 @@ class UpdatingInput extends Component<UpdatingInputProps, UpdatingInputState> {
             }
           }
         }}
-        onBlur={handleSave}
+        onBlur={(event: React.ChangeEvent<HTMLInputElement>) => {
+          if (event.target.value.length > 0) handleSave();
```

Comment:
There are cases where we may want to allow 0-length answers, so I don't think we can skip saving in this case.

It may be worth trying to check to see if the input has changed since the last save? That's probably complex enough to make a separate PR though.